### DISCUSSION TITLE
Copilot Feedback - 3 Days in

### DISCUSSION THREAD

**Discussion by srepollock** [community/community#7248] [Copilot Conversations]
Hello Copilot team,

I really enjoy the application - only 3 days in I'm seeing a great improvement with code. However, I hope there is a setting or some ability to turn off or heavily reduce the autocomplete when writing notes. This would be inline comments per language.

Overall the extension is incredible in VSCode! I hope you continue to roll out great features like copilot 💻!

**Comment by davecheney** [comment]
@srepollock you can disable copilot per file type by clicking on the copilot icon which will prompt you to disable copilot for the current file's extension, or globally.

**Reply by srepollock** [reply]
Please note that I did say 
> This would be **inline comments** per language.

**Reply by davecheney** [reply]
My apologies I misunderstood what you meant when you wrote "reduce the autocomplete when writing notes"
